src/server/helpers/search.js

```diff
@@ -358,16 +371,36 @@ export async function generateIndex(orm) {
 	}));
 	await _processEntityListForBulk(processedAreas);
 
+	const editorCollection = await Editor.forge()
+		// no bots
+		.where('type_id', 1)
+		.fetchAll();
+	const editors = editorCollection.toJSON();
+
+	/** To index names, we use aliasSet.aliases.name and bbid, which Editors don't have.
+	 * We massage the editor to return a similar format as BB entities
+	 */
+	const processedEditors = editors.map((editor) => new Object({
+		aliasSet: {
+			aliases: [
+				{name: editor.name}
+			]
+		},
+		bbid: editor.id,
```

Comment:
Do you approve of this?
By adding `bbid` here,  I avoided adding a special case for `editor` here. [search-table-row-link](https://github.com/bookbrainz/bookbrainz-site/blob/a8aa31b49e8cb9e63f9ddd71f9bf37fef528e350/src/client/components/pages/parts/search-results.js#L77)
The same can be done for collections too..